src/main/java/app/hero_maker/HeroController.java

```diff
@@ -0,0 +1,46 @@
+package app.hero_maker;
+
+import org.springframework.beans.factory.annotation.Autowired;
+    import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+import java.util.List;
+
+@Controller
+public class HeroController {
+
+    @Autowired
+    private DataValidator validator;
+
+    @InitBinder
+    private void initBinder(WebDataBinder binder){
+        binder.setValidator(validator);
+    }
+
+    @Autowired
+    private HeroDao heroDao;
+
+    @GetMapping("/heroes")
+    public @ResponseBody
+    List<Hero> getHeroes() {
+        return heroDao.getHeroes();
+    }
+
+    @PostMapping("/heroes/add")
```

Comment:
nie robi sie tak, to nie jest restowe /heroes wystarczy a niech metody same mowia czy to jest add delete itd.